### DISCUSSION TITLE
The DXXC Tab

### DISCUSSION THREAD

**Discussion by rsaue** [foldynl/QLog#143] [Ideas]
Hi @foldynl .
I am a new user of QLog. My intention is to let  it replace CQRLOG the Linux side of my two ham shacks (I am also on Win11).  I have Qlog installed on Win11 and Ubuntu 22.10. Looks great so far. 

I would suggest that you remodel the DXCC Tap matrix.  There is a lot of free space to do this in the DXXC Window. 

![DXCC tab qlog](https://user-images.githubusercontent.com/22942440/208077737-e0e9ef8b-7c4b-4760-9969-5ec5eca8f964.png)

My suggestions: 

1. Make the cells bigger. 
 
3. Alter the rows and columns  - like MODE to the left as rows and BAND on top as columns. This will make better us of all the free space. 
 
4. Let the columns show also unworked bands. This gives the operator at an instant view on his overall Status for the selected DXCC country

5. Let the user define which bands to show in the matrix, HF vs VHF/UHV etc. 

6. Place a (black) frame around the cell indicating the band and mode the rig is currently on. Gives the operator an instant indication on status of the current station on the current band and mode. 

7.  Use more colours to display more information in a cell (not worked - worked but no QSL/LOTW - Worked with QSL/LOTW

8. Let the user click on a cell to get information on which QSOs the information relates to (pop up window that shows the QSOs.

---

An example on what I am suggesting is how it is done in LogHX3 that I currently use on my windows desktop

![dxcc  loghx3](https://user-images.githubusercontent.com/22942440/208078169-f1fca555-9348-402a-9f14-e8d8c67a5e86.png)

73 de LB2EG Richard

**Comment by foldynl** [comment]
Hi Richard,

yeah, you are right. It is good idea to rework the DXCC widget. I will enqueue it to the next version.

**Reply by foldynl** [reply]
I opened an issue #144
